pkg/sensors/exec/procevents/proc_reader.go

```diff
@@ -294,8 +294,12 @@ func writeExecveMap(procs []procs) {
 		v.Namespaces.TimeChildInum = p.time_for_children_ns
 		v.Namespaces.CgroupInum = p.cgroup_ns
 		v.Namespaces.UserInum = p.user_ns
+		v.BinaryPath.PathLength = int64(copy(v.BinaryPath.Path[:], p.exe))
 
-		m.Put(k, v)
+		err := m.Put(k, v)
+		if err != nil {
+			logger.GetLogger().WithError(err).Fatal("Could not put value in execve_map")
```

Comment:
Do we need to crash the agent here?
That's typically a bad idea (tm)

Reply:
yep done as a warning, I also added correct fields to the logging.